### ISSUE TITLE
Scout cloak should deactivate on death

### ISSUE THREAD

**Issue by dasWurmtMich** [tgstation/TerraGov-Marine-Corps#355]
If you die with your scout cloak on pretty much no one is gonne see you and everyone is just gonne walk over you while you are dead

**Comment by DominikPanic** [comment]
Not getting properly processed @Surrealaser 
```
/obj/item/storage/backpack/marine/satchel/scout_cloak/proc/process_camo_cooldown(mob/living/user, cooldown)
	to_chat(world, "Processing camo cooldown")
```
![dreamseeker_2019-01-27_15-39-29](https://user-images.githubusercontent.com/6252677/51802393-f7641880-2249-11e9-814e-fc8e0df706cf.png)

**Comment by DominikPanic** [comment]
FIxed in #499
